admin/tests/test_securedrop-admin.py

```diff
@@ -287,6 +287,26 @@ def test_validate_gpg_key(self, caplog):
             site_config.validate_gpg_keys()
         assert 'FAIL does not match' in e.value.message
 
+    def test_validate_sasl_domain(self, caplog):
+        args = argparse.Namespace(site_config='INVALID',
+                                  ansible_path='tests/files',
+                                  app_path=dirname(__file__))
+        site_config = securedrop_admin.SiteConfig(args)
+        site_config.config = {
+            'ossec_alert_gpg_public_key':
+            'test_journalist_key.pub',
+
+            'ossec_gpg_fpr':
+            '65A1B5FF195B56353CC63DFFCC40EF1228271441',
+
+            'sasl_username':
+            'alice',
+
+            'sasl_domain':
+            '',
+        }
+        assert site_config.ValidateSASLDomain()
+
```

Comment:
Since there is no domain name validation at the moment, there is no need for a dedicated validation test function. The consistency check of the `sasl_domain` declaration above are taken care of by:

<pre>
verify_prompt_sasl_domain = verify_desc_consistency
</pre>

below